releaf-core/spec/lib/error_formatter_spec.rb

```diff
@@ -60,34 +67,38 @@ def base_validation
     end
 
     it "correctly adds errors for fields resource fields" do
-      expect( subject.errors["resource[title]"] ).to eq [blank_error]
+      expect( subject.errors["resource[title]"] ).to eq [blank_error('title')]
     end
 
     it "correclty adds errors for missing associated object (belongs_to)" do
-      expect( subject.errors["resource[author_id]"] ).to eq [blank_error]
+      expect( subject.errors["resource[author_id]"] ).to eq [blank_error('author')]
```

Comment:
Line is too long. [83/80]<br>Space inside parentheses detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.